### ISSUE TITLE
UI enhancements

### ISSUE THREAD

**Issue by dquinton** [mifi/lossless-cut#431]
I notice there's some great improvement been made to LLcut recently, well done for all the work, its a very useful tool to have available.  The changing cut modes is very handy.

OK, some feedback ideas on the UI...

1. Keep/Remove modes 
I think the popup at top when switching modes is too wordy, how about just:

Mode: Keep Selections
Mode: Trash Selections

But seeing that there is a tool-tip on the mode icon, perhaps the popup is not needed?
The trash and Save icons that show on the timeline make it pretty clear what mode one is in.

2. Dialogues 
These all have white bg and this is in contrast to the app UI (for me) which is dark.
(I mean the Settings window, merge tool and others)

3. Processing
When writing any file there is a rainbow thing - what the??  I think a progress bar would be more aesthetic/assuring.

thanks

**Comment by mifi** [comment]
Thx for suggestions.
1. The reason I'm so verbose with these messages is because many new people don't understand exactly how this works, so I kept repeating myself.
2. Yea I know, just the UI library I'm using doesn't look good on or support dark.
3. It's magic 🌈

**Comment by mifi** [comment]
Can continue here #189 - closing this issue
